lang/cs/Org.Apache.REEF.Wake/Time/Runtime/RuntimeClock.cs

```diff
@@ -168,7 +168,7 @@ public void Run()
             }
             catch (Exception e)
             {
-                runtimeException = Optional<Exception>.Of(new SystemException("Caught Exception in clock, failing the Evaluator.", e));
+                runtimeException = Optional<Exception>.Of(new WakeRuntimeException("Caught Exception in clock, failing the Evaluator.", e));
```

Comment:
Since this is in Wake, why not use the already existing WakeRuntimeException, which is more specific than ReefRuntimeException?